modules/__tests__/Router-test.js

```diff
@@ -272,7 +268,7 @@ describe('Router', function () {
         render() {
           const { label, children } = this.props
           const child = React.cloneElement(children, {
-            createElement: this.createElement
+            createElement: this.createElement.bind(this)
```

Comment:
Nit - please bind this in the constructor. Even though this is in test code, I'd rather not users accidentally follow this pattern without recognizing the consequences.